### PR TITLE
Add volume control keyboard shortcuts

### PR DESCRIPTION
**Pull request by delthas**
:pleading_face:

### CODE REVIEW

File: player/mpv.go [hunk 1]
```diff
@@ -204,6 +205,15 @@ func (mpv *MPV) setProperty(name, value string) {
 	mpv.checkError(C.mpv_set_property_async(mpv.handle, 1, cName, C.MPV_FORMAT_STRING, unsafe.Pointer(&cValue)))
 }
 
+func (mpv *MPV) SetVolume(volume float64) {
```
Comment (DucNg) [inline player/mpv.go:208]: There is already a setVolume function line 280, it's just commented out :laughing: 

Maybe try to reuse it, or just delete it if it isn't suitable. It's probably a bad sign if it's commented.

File: gui/list.go [hunk 2]
```diff
@@ -52,6 +54,7 @@ func UpdateMetadataLoop(m *model, delayToRefresh time.Duration) {
 
 func (m *model) Init() tea.Cmd {
 	m.playingItemIndex = m.list.Index()
+	m.volume = 100
```
Comment (DucNg) [inline gui/list.go:57]: We should consider restoring previous volume on startup. Same goes mpv.Initialize() function, I forced the default volume to 100 but we should consider handling it properly if we're implementing the feature.

**Review by DucNg — COMMENTED** [review]
I deliberately didn't bother with volume (since it can be changed for individual applications via PulseAudio volume control). But if you really want to implement this feature I think we should consider restoring volume on startup. If we don't sound might be surprisingly loud when quitting and rerunning the app.

If you use pulseaudio to change the volume of the individual app, it is saved even if the application is closed. Maybe it fits your use case?

If that's too bothering I don't mind if you do it in a second PR.

File: gui/list.go (hunk 2 above)
Comment (delthas) [reply gui/list.go:57]: Can be done in a separate MR 😬 🤡

**Comment by delthas** [comment]
Updated. 💟

**Pull request merged**